src/lib/Components/Consignments/Submission/Gemini/createGeminiAssetWithS3Credentials.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  createGeminiAssetWithS3CredentialsMutation,
+  CreateGeminiEntryForAssetInput,
+} from "__generated__/createGeminiAssetWithS3CredentialsMutation.graphql"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { commitMutation, graphql } from "relay-runtime"
+
+export const createGeminiAssetWithS3Credentials = (input: CreateGeminiEntryForAssetInput) => {
```

Comment:
I like how you can use Relay (and tooling) and not need to be in the context of an actual Relay container. ie- you can use it just in helpers/utils too